src/utils/error-utils.js

```diff
@@ -15,10 +15,22 @@ import { StatusCodes } from 'http-status-codes'
  */
 
 export const failAction = (_request, h, err) => {
-  const errors = err.details.map((detail) => ({
-    message: detail.message,
-    property: detail.path[0],
-    value: err._original[detail.path[0]]
-  }))
-  return h.response({ errors }).code(StatusCodes.BAD_REQUEST).takeover()
+  const formattedErrors =
+    err?.details?.map((detail) => {
+      const detailPath =
+        detail.path && detail.path.length > 0 ? detail.path[0] : undefined
+
+      return {
+        message: detail.message,
+        property: detail?.context?.path || detailPath,
+        value:
+          detail?.context?.value ||
+          (err._original && detailPath ? err._original[detailPath] : undefined)
```

Comment:
I think this falls foul of our rules on combining ternaries and AND / OR expressions so would be clearer if value was set above or possibly in a helper function: https://eaflood.atlassian.net/wiki/spaces/IWTF/pages/4295622851/Common+Code+Review+Issues (towards the bottom of the section 'Code Style')